apps/lcs/behaviours/locals-enricher.js

```diff
@@ -0,0 +1,13 @@
+module.exports = superclass => class extends superclass {
+  getErrorMessage(error, req, res) {
+    const locals = {};
+    const valuesEnriched = req.sessionModel.get('valuesEnriched');
+
+    if (valuesEnriched && Object.keys(valuesEnriched).length > 0) {
```

Comment:
Is it worth protecting against cases where this object is empty (maybe not possible now but in future?)

Reply:
This statement will first check that `valuesEnriched' is true, i.e. it will fail if value is null, undefined, '', 0

if it's [] or {}.
Object.keys(valuesEnriched).length will return 0